drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -252,4 +252,7 @@ config NRF700X_RPU_PS_IDLE_TIMEOUT_MS
 	default 10
 
 endif
+config NRF700X_RSSI_STALE_TIMEOUT_MS
+	int "RSSI stale timeout in milliseconds"
+	default 100
```

Comment:
Should the default be higher, say `1000` for ping case? Only when the STA is highly mobile this needs to be lower (and this should be explained in help).